src/modules/lifecycle_management.json

```diff
@@ -0,0 +1,254 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "LifecycleManagement",
+        "description": "A module for managing application lifecycle.",
+        "version": "0.0.0"
+    },
+    "components": {
+        "schemas": {
+            "State": {
+                "type": "string",
+                "description": "The application lifecycle state",
```

Comment:
Might be worth sharing this enum with the Core SDK somehow

Reply:
How is this done for other components, e.g. KeyboardType, pinSpace, etc.?